cg-producer/pycg_producer/producer.py

```diff
@@ -275,6 +268,11 @@ def _get_lines_of_code(self, files_list):
         return res
 
 
+    def _get_max_rss(cmd):
+        benchmark_results = cmdbench.benchmark_command(' '.join(map(str, cmd)))
```

Comment:
Would this not run the `cmd` a second time? There is also an execute on line 243.

Reply:
Yes, sorry, it totally slipped me out. I replaced the subprocess command on line 243 with the execution of the benchmark command.